### ISSUE TITLE
I have a few question!

### ISSUE THREAD

**Issue by khoaprogrammer** [experiencor/basic-yolo-keras#30]
1.  I'm working on a school project about hand detection and i'm trying to use your model with this dataset:
http://www.robots.ox.ac.uk/~vgg/data/hands/. The problem is the dataset is too large and i don't know if this dataset can work with this model yet so i want to make sure that this dataset works. What i'm trying to do is to split the dataset to a number of chunks (about 50 images each) and train the model chunk by chunk to see if the prediction is better after each chunk. Is this a good approach? Because i dont have much time to play with this. 

3. Which value of val_loss is at least enough to make a good prediction?

2. I see that you have used this model to make hand prediction, can you give me the pretrained weight?

Thank you!

**Comment by experiencor** [comment]
1. Hmm... If you don't have time to train the whole dataset, I would suggest you to just use a portion of the dataset for training and validation. Says, put 100 images in the training folder and 20 images in the validation folder. And by chunk, you mean batch? The current code reads the dataset in batches, says 10 images at a time. The batch size can be specified in the config.json file.

2. I would say 1.2 or something. Bigger models (Full Yolo, Inception3) would be able to drive the loss lower.

3. Only the result is shared by the author. I will update if he want to share the weight.

**Comment by khoaprogrammer** [comment]
1. No, it's not batch, it's just split dataset to multiple parts. For example, i have 1000 images, and i don't have time to train that much in one time (and see it fail!). So i split it about 50 images per part. Today, i train it with the first part and after that i see that it doesn't predict well (maybe not enough information), so i continue to train it with the second part and check that if the model can learn well with more information. That is what i think because i'm not good at this.

2. That's weird! i use full yolo model (400 pic to train and 100 more to test) and the val_loss is about 0.2 after 2 or 3 epochs. But the model was fail to predict, even if i use an image from the train set to test.

3. 👍 

4. Can i have a more direct way to contact with you? I'm a Vietnamese too and i have so many question about this model!

**Comment by experiencor** [comment]
My email is experiencor@gmail.com. Email me your facebook.

**Comment by Kirthi1212** [comment]
Hi I have the exact same issue. I used tiny yolo to train on the hand dataset. the val_loss comes down to 0.2 after 3 epochs. But the model is not predicting anything

**Comment by Kirthi1212** [comment]
>     1. No, it's not batch, it's just split dataset to multiple parts. For example, i have 1000 images, and i don't have time to train that much in one time (and see it fail!). So i split it about 50 images per part. Today, i train it with the first part and after that i see that it doesn't predict well (maybe not enough information), so i continue to train it with the second part and check that if the model can learn well with more information. That is what i think because i'm not good at this.
> 
>     2. That's weird! i use full yolo model (400 pic to train and 100 more to test) and the val_loss is about 0.2 after 2 or 3 epochs. But the model was fail to predict, even if i use an image from the train set to test.
> 
>     3. +1
> 
>     4. Can i have a more direct way to contact with you? I'm a Vietnamese too and i have so many question about this model!

Hi I have the exact same issue. I used tiny yolo to train on the hand dataset. the val_loss comes down to 0.2 after 3 epochs. But the model is not predicting anything. How did you solve this?
